src/main/webapp/popup.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Opens a popup window for adding an event when the add-button is pressed
+ */
+function openEventPopup() {
+    document.getElementById('add-event').style.display = 'block';
```

Comment:
So far it is ok, but the element shouldn't be hard coded but taken from event from example.

Reply:
I added TODO for adding a class in the future